pkg/operator/vspherecontroller/driver_starter.go

```diff
@@ -134,6 +134,11 @@ func (c *VSphereController) createCSIDriver() {
 			metricsCertSecretName,
 			c.apiClients.SecretInformer,
 		),
+		csidrivercontrollerservicecontroller.WithSecretHashAnnotationHook(
+			defaultNamespace,
+			driverConfigSecretName,
```

Comment:
@bertinatto I had to fix the order of namespace / secret name, it's different than in `WithSecretDaemonSetAnnotationHook`
PTAL again

/hold cancel